### PR TITLE
Add delete user for admin console

### PR DESCRIPTION
**Pull request by zatkiller**
- Add delete user route to support delete admin functionality
- Some edits to support admin console admins tab

### CODE REVIEW

File: src/controllers/userController.ts [hunk 1]
```diff
@@ -223,11 +223,34 @@ async function updateUserPassword(req: Request, res: Response, next: NextFunctio
...
+    if (!user) {
+      throw new NotFoundError('User cannot be found');
+    }
+    req.user = user;
+    next();
+  } catch (err) {
+    next(err);
+  }
+}
+async function destroyUser(req: Request, res: Response, next: NextFunction) {
```
Comment (ivantjh) [inline src/controllers/userController.ts:238]: New line here

File: src/controllers/userController.ts [hunk 2]
```diff
@@ -137,7 +137,7 @@ async function retrieveUserByEmail(req: Request, res: Response, next: NextFuncti
       //for FE to get the user Id to send a POST request during validation
 
       attributes: {
-        exclude: ['password', 'role', 'username'],
+        exclude: ['password', 'role'],
```
Comment (ivantjh) [inline src/controllers/userController.ts:140]: Comment above has typos

**Review by ivantjh — APPROVED** [review]
lgtm!

File: src/controllers/userController.ts (hunk 2 above)
Comment (zatkiller) [reply src/controllers/userController.ts:140]: Fix in PR #72

File: src/controllers/userController.ts (hunk 1 above)
Comment (zatkiller) [reply src/controllers/userController.ts:238]: Fix in PR #72

**Pull request merged**